File: truth_set/javascript/resource/variable-initialisation/compareScores.js
function compareScores() {
    var playerScore;
    var threshold = 50;
    var assignScore = Math.random() > 0.5;
    if (assignScore) {
        playerScore = 75;
    }
    if (playerScore > threshold) {
        console.log("Player has a high score!");
    }
    else {
        console.log("Player score is below the threshold.");
    }
}
compareScores();

File: truth_set/python/resource/variable-initialisation/compareScores.py
def compare_scores():
    player_score = None
    threshold = 50
    assign_score = True if __import__('random').random() > 0.5 else False
    
    if assign_score:
        player_score = 75
        
    if player_score > threshold:  
        print("Player has a high score!")
    else:
        print("Player score is below the threshold.")

compare_scores()